apps/src/templates/instructions/MarkdownInstructions.jsx

```diff
@@ -53,6 +55,22 @@ const MarkdownInstructions = React.createClass({
       });
     }
 
+    if (this.props.isBlockly) {
+      // Convert any inline XML into blockly blocks. Note that we want to
+      // make sure we don't initialize any blockspace before the main
+      // block space has been created, lest we violate some assumptions
+      // blockly has.
```

Comment:
Do note that this is only necessary for blocks that are needed at page load time; anything that's dependent on a user action will be fine as is.